cdap-proto/src/main/java/co/cask/cdap/proto/artifact/preview/PreviewConfig.java

```diff
@@ -33,13 +33,15 @@
   private final ProgramType programType;
   private final Set<String> realDatasets;
   private final Map<String, String> runtimeArgs;
+  private final Integer runningTime;
```

Comment:
Yes. Since for batch pipeline it will get deleted after 15 mins of run, so I think we should apply this to batch also.